File: finger_exercises/primes_to_1000.c
#include <stdio.h>

int main(void){
    int n;
    int stop = 1000;
    int summation = 0;
    int j;
    int counter;

    for (n = 2; n < stop; n++){
        if (n % 2 != 0){
            counter = 0;
            for (j = 2; j < n; j++)
            {
                if (n % j == 0){
                    counter++;
                    break;
                }
            }
            if (counter == 0){
                summation += n;
            }
            
        }
    }
    printf("Sum = %d: ", (summation + 2));
}

File: finger_exercises/primes_to_1000.py
"""Write a program that prints the sum of the prime numbers greater than 2
and less than 1000."""

summation = 0
for num in range(2, 1000):
    if num % 2 != 0:
        is_prime = True
        for i in range(2, num):
            if num % i == 0:
                is_prime = False
                break
        if is_prime == True:
            summation += num
# 2 is a prime number            
print(summation + 2)
print()
